src/handlers/ExecuteViewSubmitHandler.ts

```diff
@@ -0,0 +1,54 @@
+import { IHttp, IModify, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { IRoom } from '@rocket.chat/apps-engine/definition/rooms';
+import { UIKitViewSubmitInteractionContext , UIKitInteractionContext} from '@rocket.chat/apps-engine/definition/uikit';
+import { ModalsEnum } from '../enums/Modals';
+import { sendMessage } from '../lib/message';
+import { ClickUpApp } from '../../ClickUpApp';
+import { postTask } from '../lib/postTask';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
+import { getUIData } from '../lib/persistence';
+
+export class ExecuteViewSubmitHandler {
+	constructor(
+		private readonly app: ClickUpApp,
+		private readonly read: IRead,
+		private readonly http: IHttp,
+		private readonly modify: IModify,
+		private readonly persistence: IPersistence
+        
+	) {}
+
+	public async run(context: UIKitViewSubmitInteractionContext,read: IRead,
```

Comment:
Maybe you may consider encapsulating the whole method in a try-catch, so there's will no need for multiple try-catches along with every case on the switch

Reply:
Done, sir.